Patch to add the Gentoo-prefix to the library and include search path.

Note:
The variable GENTOO_GCC_INCLUDE needs to be set in the preinstallopts inside the easyconfig.
We can't set it directly inside of this Template, otherwise the path gets mangled, because
some components of it happen do have the same name as some define statements.

author: Oliver Stueker (ACENET)
date:   2023-12-12
diff --git a/config/Template b/config/Template
index a5f1e38..7efcd42 100644
--- a/config/Template
+++ b/config/Template
@@ -620,8 +620,8 @@ PYTHONINCSEARCH		= PythonIncSearch
 PYTHONPKGSDIR           = PythonPkgsDir
 PYTHONBINDIR            = PythonBinDir
 
-LIB_SEARCH		= $(LIBSEARCH) ExtraLibSearch
-INC_SEARCH		= $(INCSEARCH) ExtraIncSearch
+LIB_SEARCH		= $(LIBSEARCH) ExtraLibSearch  -L$(EBROOTGENTOO)/lib64
+INC_SEARCH		= $(INCSEARCH) ExtraIncSearch  -I$(GENTOO_GCC_INCLUDE)  -I$(EBROOTGENTOO)/include
 
 TOP			= TOPDIR
 CURRENT_DIR		= CURDIR
